include/cuco/detail/open_addressing/open_addressing_ref_impl.cuh

```diff
@@ -111,7 +111,8 @@ class open_addressing_ref_impl {
 
   static constexpr auto cg_size = probing_scheme_type::cg_size;  ///< Cooperative group size
   static constexpr auto window_size =
-    storage_ref_type::window_size;  ///< Number of elements handled per window
+    storage_ref_type::window_size;             ///< Number of elements handled per window
+  static constexpr auto thread_scope = Scope;  ///< CUDA thread scope
```

Comment:
shall we use `thread_scope` consistently across the base ref class?

Reply:
I tried to do that e.g. when we dispatch the CUDA-C atomics in the CAS funtion but I ran into some errors. I didn't dive deeper into it but it might have to do with the mixin stuff since inside the CAS we don't cast *this to the ref class and so theead_scope might not be available.

Reply:
AFAIK there no other occurrences of Scope in the OA implementation

Reply:
I recall I ran into similar issues as well.